lib/kennel/models/project.rb

```diff
@@ -4,7 +4,7 @@ module Models
     class Project < Base
       settings :team, :parts, :tags, :slack
       defaults(
-        tags: -> { ["project:#{kennel_id}"] + team.tags },
+        tags: -> { Array(["project:#{kennel_id}"] + team.tags).uniq },
```

Comment:
Still need to do this in the subclasses, so this commit is probably unnecessary.

Reply:
don't need the `Array`